pkg/policy/condition/func.go

```diff
@@ -162,6 +167,11 @@ func (functions *Functions) UnmarshalJSON(data []byte) error {
 	return nil
 }
 
+// GobDecode - decodes gob data to Functions.
+func (functions *Functions) GobDecode(data []byte) error {
+	return functions.UnmarshalJSON(data)
+}
+
```

Comment:
Its better to keep encoder/decoder together

Reply:
> Its better to keep encoder/decoder together to keep method name sorted.

Done